.pre-commit-config.yaml

```diff
@@ -0,0 +1,21 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.5.0
+    hooks:
+    -   id: check-toml
+    -   id: end-of-file-fixer
+    -   id: trailing-whitespace
+    -   id: check-case-conflict
+    -   id: check-executables-have-shebangs
+    -   id: check-shebang-scripts-are-executable
+    -   id: check-symlinks
+    -   id: check-vcs-permalinks
+    -   id: destroyed-symlinks
+    -   id: detect-private-key
+    -   id: fix-byte-order-marker
+    -   id: mixed-line-ending
+
+-   repo: https://github.com/macisamuele/language-formatters-pre-commit-hooks
+    rev: v2.12.0
+    hooks:
+    -   id: pretty-format-rust
```

Comment:
Not sure how this works so it would be nice to mention this in the README

Reply:
Oops. This should be here. Those are my local pre-commit checks. I don't think we plan to use `pre-commit` for this project, so I will remove this file 😅